src/commonMain/kotlin/com/monta/slack/notifier/SlackClient.kt

```diff
@@ -126,81 +121,54 @@ class SlackClient(
         messageId: String? = null,
         previousAttachments: List<SlackMessage.Attachment>? = null,
     ): SlackMessage {
-        val attachments = mutableMapOf<JobType, SlackMessage.Attachment>()
+        val attachments = if (appendAttachments) {
```

Comment:
This is the primary change of the PR, adding support for appending attachments instead of replacing them, depending on the input property.

Reply:
I think this would break the updating of status in the notifier?

<img width="553" alt="image" src="https://github.com/user-attachments/assets/a8977836-532d-47c3-9384-b1ed22f9afca">

Specifically where it goes from `In Progress` to `Success` or am I wrong?